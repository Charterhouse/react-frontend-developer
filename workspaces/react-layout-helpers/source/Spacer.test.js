import React from 'react'
import { render } from 'react-testing-library'
import { Spacer } from './'

describe('Spacer', () => {
  it('has proper styling', () => {
    const { container } = render(<Spacer>test</Spacer>)
    expect(container).toMatchSnapshot()
  })

  it('has proper styling when using render prop', () => {
    const { container } = render(<Spacer render={() => 'test'} />)
    expect(container).toMatchSnapshot()
  })
})
