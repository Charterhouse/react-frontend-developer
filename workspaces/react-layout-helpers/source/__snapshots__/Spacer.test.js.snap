// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer has proper styling 1`] = `
<div>
  <div
    style="margin: 0px; padding: 0px; box-sizing: border-box;"
  >
    test
  </div>
</div>
`;

exports[`Spacer has proper styling when using render prop 1`] = `
<div>
  <div
    style="margin: 0px; padding: 0px; box-sizing: border-box;"
  >
    test
  </div>
</div>
`;
